ueberdosis/tiptap
# fix: Update peerDependencies to fix lerna version tasks

## Please describe your changes

This PR fixes issues we ran into with lerna not being able to update our package-lock because of invalid peerDependencies. This PR solves this issue.

It also includes a change for the versioning task. The version task now won't push and commit automatically anymore. I think it's a good idea, because that way the developer can:

1. Verify the changes made by Lerna
2. Test versioning and packaging updates
3. Have control over the tagging process

Sure makes things a bit more complicated but also allows for more fine grained control.

## How did you accomplish your changes

I went through all packages one by one and replaced the peerDependencies to their correct versions.

## How have you tested your changes

I created local test versions including new package-lock files and it worked out fine. bumped up to `2.1.0-pre.0` for a test.

## How can we verify your changes

Clone my pull request and run `npm run release:minor:pre` and see if the versions + the package-lock are updated.

## Remarks

Nothing to add here.

## Checklist


## Related issues

No related issues.
